### PR TITLE
[CosmosDB] Renaming all references to autopilot

### PR DESCRIPTION
**Pull request by ravgill**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by MehaKaushik — APPROVED** [review]

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json [hunk 1]
```diff
@@ -5900,11 +5904,54 @@
...
+          "$ref": "#/definitions/ThroughputPolicyResource"
+        }
+      }
+    },
+    "ThroughputPolicyResource": {
+      "type": "object",
+      "description": "Cosmos DB resource throughput policy",
+      "properties": {
+        "isEnabled": {
+          "type": "boolean",
```
Comment (sanjaiganesh) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: specify default ?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json [hunk 2]
```diff
@@ -6545,6 +6592,12 @@
     },
     "CreateUpdateOptions": {
       "type": "object",
+      "properties": {
+        "throughput": {
+          "type": "string",
```
Comment (sanjaiganesh) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: regex to ensure the format ?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json [hunk 3]
```diff
@@ -5900,11 +5904,54 @@
...
         }
+      }
+    },
+    "ProvisionedThroughputSettingsResource": {
+      "type": "object",
+      "description": "Cosmos DB provisioned throughput settings object",
+      "properties": {
+        "maxThroughput": {
+          "type": "integer",
+          "description": "Represents maximum throughput container can scale up to."
```
Comment (sanjaiganesh) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: configure max value ?

Same for targetMaxThroughput

**Review by sanjaiganesh — COMMENTED** [review]
pls address the comments.

**Comment by ravgill** [comment]
@MyronFanQiu / @markcowl / @yungezz can someone review this PR?

**Comment by yungezz** [comment]
@ravgill although ARM signed off, this PR will cause SDK breaking change since renaming properties in existing stable version.

**Review by markcowl — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
@yungezz Yes, just found out that it breaked Fluent .NET SDK :-(

**Pull request merged**